components/MeetingDetail/MeetingDetail.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import DogName from "../DogName/DogName";
+import ImageWithText from "../ImageWithText/ImageWithText";
+
+export default function MeetingDetail({ data }) {
+  const { location, date, time, excluded, furtherInfo, attending } = data;
+  return (
+    <StyledArticle>
+      <h2>{location}</h2>
+      <ImageWithText
+        image="calendar-icon.svg"
+        altText="Kalender Icon"
+        text={date}
+      />
+      <ImageWithText image="clock-icon.svg" altText="Uhr Icon" text={time} />
+      {excluded ? (
+        <ImageWithText
+          data-testid="excluded"
+          image="stop-icon.svg"
+          altText="Stop Icon"
+          text={excluded}
+        />
+      ) : null}
+      <StyledInfobox>
+        {furtherInfo ? (
+          <div data-testid="info">
+            <label htmlFor="weitereInfos">Weitere Infos:</label>
+            <StyledParagraph id="weitereInfos">{furtherInfo}</StyledParagraph>
+          </div>
+        ) : null}
+      </StyledInfobox>
+      <StyledInfobox>
+        <label htmlFor="teilnehmende">Wir sind dabei:</label>
+        <DogName id="teilnehmende" attendingDog={attending} />
```

Comment:
When there will be more attending dogs, how about to use "ul" and "li" elements? 😆

Reply:
I like the idea! It makes total sense but it is not a part of my user story for now. I will integrate it in my US3, when you can add attendees via form input